dev/com.ibm.websphere.security/src/com/ibm/websphere/security/audit/AuditEvent.java

```diff
@@ -114,6 +114,15 @@
     public static final String TARGET_EJB_BEAN_NAME = "target.ejb.beanname";
     public static final String TARGET_EJB_METHOD_PARAMETERS = "target.ejb.method.parameters";
 
+    public static final String TARGET_SAF_RETURN_CODE = "target.saf.return.code";
+    public static final String TARGET_RACF_RETURN_CODE = "target.racf.return.code";
+    public static final String TARGET_RACF_REASON_CODE = "target.racf.reason.code";
+    public static final String TARGET_USER_SECURITY_NAME = "target.user.security.name";
+    public static final String TARGET_APPLID = "target.applid";
+    public static final String TARGET_SAF_PROFILE = "target.saf.profile";
+    public static final String TARGET_SAF_CLASS = "target.saf.class";
+    public static final String TARGET_AUTHORIZATION_DECISION = "target.authorization.decision";
```

Comment:
I guess I am questioning some of these constants... wouldn't the `target.authorization.decision` be the same as the `outcome` which is defined below?  Should we be using some standard way of indicating things like the auth decision, and the return/reason codes?  @emilytee